v2/rscg_examples/IDisposableGenerator/src/IDisp/ConnectionDB.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace IDisposableGeneratorDemo;
+
+class ConnectionDB : IDisposable
```

Comment:
This can also use the generator as well, there is a feature in it to tell it to call methods inside of it as well.